.github/templates/linux_ci_workflow.yml.in

```
@@ -32,10 +32,15 @@ jobs:
     outputs:
       docker_image: ${{ steps.calculate-tag.outputs.docker_image }}
     steps:
+      - name: Log in to ECR
+        run: |
+          aws ecr get-login --no-include-email --region us-east-1 > /tmp/ecr-login.sh
```

Please use `aws ecr get-login-password`, as get-login was removed in awscli-2.x 
```suggestion
          aws ecr get-login-lassword --region us-east-1| docker --username AWS --password-stdin "308535385114.dkr.ecr.us-east-1.amazonaws.com
```

---

sure, also going to refactor (it was duplicated in multiple places) and add a few code comments in the following PR